packages/adp-tooling/templates/rta/common/custom-action.xml

```diff
@@ -0,0 +1,5 @@
+<!-- Use stable and unique IDs!-->
+<core:FragmentDefinition xmlns:core='sap.ui.core' xmlns='sap.m'>
+    <!--  add your xml here -->
+    <Button text="New Button"  id="<%- ids.toolbarActionButton %>"></Button>
```

Comment:
just in case there is two white spaces `text="New Button"  id`, not sure if it is intended